2021-09-16 11:21:38,923 INFO    MainThread:117553 [internal.py:wandb_internal():89] W&B internal server running at pid: 117553, started at: 2021-09-16 11:21:38.923285
2021-09-16 11:21:38,925 DEBUG   HandlerThread:117553 [handler.py:handle_request():126] handle_request: check_version
2021-09-16 11:21:38,925 INFO    WriterThread:117553 [datastore.py:open_for_write():77] open: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/JanataHack-NLP-Hackathon/wandb/run-20210916_112128-20zmzt4y/run-20zmzt4y.wandb
2021-09-16 11:21:38,927 DEBUG   SenderThread:117553 [sender.py:send():182] send: header
2021-09-16 11:21:38,927 DEBUG   SenderThread:117553 [sender.py:send_request():196] send_request: check_version
2021-09-16 11:21:39,257 DEBUG   SenderThread:117553 [sender.py:send():182] send: run
2021-09-16 11:21:39,259 DEBUG   SenderThread:117553 [git.py:repo():33] git repository is invalid
2021-09-16 11:21:40,178 DEBUG   HandlerThread:117553 [handler.py:handle_request():126] handle_request: run_start
2021-09-16 11:21:40,179 INFO    SenderThread:117553 [dir_watcher.py:__init__():169] watching files in: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/JanataHack-NLP-Hackathon/wandb/run-20210916_112128-20zmzt4y/files
2021-09-16 11:21:40,179 INFO    SenderThread:117553 [sender.py:_start_run_threads():743] run started: 20zmzt4y with start time 1631771498
2021-09-16 11:21:40,179 DEBUG   SenderThread:117553 [sender.py:send():182] send: summary
2021-09-16 11:21:40,179 INFO    SenderThread:117553 [sender.py:_save_file():873] saving file wandb-summary.json with policy end
2021-09-16 11:21:40,181 DEBUG   HandlerThread:117553 [meta.py:__init__():40] meta init
2021-09-16 11:21:40,181 DEBUG   HandlerThread:117553 [meta.py:__init__():54] meta init done
2021-09-16 11:21:40,181 DEBUG   HandlerThread:117553 [meta.py:probe():211] probe
2021-09-16 11:21:40,187 DEBUG   HandlerThread:117553 [git.py:repo():33] git repository is invalid
2021-09-16 11:21:40,187 DEBUG   HandlerThread:117553 [meta.py:_save_code():90] save code
2021-09-16 11:21:40,187 WARNING HandlerThread:117553 [meta.py:_save_code():92] unable to save code -- program entry not found
2021-09-16 11:21:40,187 DEBUG   HandlerThread:117553 [meta.py:_save_pip():58] save pip
2021-09-16 11:21:40,188 DEBUG   HandlerThread:117553 [meta.py:_save_pip():72] save pip done
2021-09-16 11:21:40,188 DEBUG   HandlerThread:117553 [meta.py:_save_conda():79] save conda
2021-09-16 11:21:41,182 INFO    Thread-8  :117553 [dir_watcher.py:_on_file_created():217] file/dir created: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/JanataHack-NLP-Hackathon/wandb/run-20210916_112128-20zmzt4y/files/requirements.txt
2021-09-16 11:21:41,182 INFO    Thread-8  :117553 [dir_watcher.py:_on_file_created():217] file/dir created: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/JanataHack-NLP-Hackathon/wandb/run-20210916_112128-20zmzt4y/files/conda-environment.yaml
2021-09-16 11:21:41,182 INFO    Thread-8  :117553 [dir_watcher.py:_on_file_created():217] file/dir created: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/JanataHack-NLP-Hackathon/wandb/run-20210916_112128-20zmzt4y/files/wandb-summary.json
2021-09-16 11:21:45,185 INFO    Thread-8  :117553 [dir_watcher.py:_on_file_created():217] file/dir created: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/JanataHack-NLP-Hackathon/wandb/run-20210916_112128-20zmzt4y/files/output.log
2021-09-16 11:21:47,476 DEBUG   HandlerThread:117553 [meta.py:_save_conda():87] save conda done
2021-09-16 11:21:47,476 DEBUG   HandlerThread:117553 [meta.py:probe():249] probe done
2021-09-16 11:21:47,500 DEBUG   HandlerThread:117553 [handler.py:handle_request():126] handle_request: stop_status
2021-09-16 11:21:47,500 DEBUG   SenderThread:117553 [sender.py:send_request():196] send_request: stop_status
2021-09-16 11:21:48,186 INFO    Thread-8  :117553 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/JanataHack-NLP-Hackathon/wandb/run-20210916_112128-20zmzt4y/files/output.log
2021-09-16 11:21:48,187 INFO    Thread-8  :117553 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/JanataHack-NLP-Hackathon/wandb/run-20210916_112128-20zmzt4y/files/conda-environment.yaml
2021-09-16 11:21:48,187 INFO    Thread-8  :117553 [dir_watcher.py:_on_file_created():217] file/dir created: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/JanataHack-NLP-Hackathon/wandb/run-20210916_112128-20zmzt4y/files/wandb-metadata.json
2021-09-16 11:21:49,127 DEBUG   SenderThread:117553 [sender.py:send():182] send: files
2021-09-16 11:21:49,127 INFO    SenderThread:117553 [sender.py:_save_file():873] saving file wandb-metadata.json with policy now
2021-09-16 11:21:49,852 DEBUG   SenderThread:117553 [sender.py:send():182] send: history
2021-09-16 11:21:49,853 DEBUG   SenderThread:117553 [sender.py:send():182] send: summary
2021-09-16 11:21:49,853 INFO    SenderThread:117553 [sender.py:_save_file():873] saving file wandb-summary.json with policy end
2021-09-16 11:21:49,854 DEBUG   SenderThread:117553 [sender.py:send():182] send: history
2021-09-16 11:21:49,855 DEBUG   SenderThread:117553 [sender.py:send():182] send: summary
2021-09-16 11:21:49,855 INFO    SenderThread:117553 [sender.py:_save_file():873] saving file wandb-summary.json with policy end
2021-09-16 11:21:50,021 DEBUG   SenderThread:117553 [sender.py:send():182] send: history
2021-09-16 11:21:50,021 DEBUG   SenderThread:117553 [sender.py:send():182] send: summary
2021-09-16 11:21:50,021 INFO    SenderThread:117553 [sender.py:_save_file():873] saving file wandb-summary.json with policy end
2021-09-16 11:21:50,054 DEBUG   SenderThread:117553 [sender.py:send():182] send: history
2021-09-16 11:21:50,054 DEBUG   SenderThread:117553 [sender.py:send():182] send: summary
2021-09-16 11:21:50,055 INFO    SenderThread:117553 [sender.py:_save_file():873] saving file wandb-summary.json with policy end
2021-09-16 11:21:50,187 INFO    Thread-8  :117553 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/JanataHack-NLP-Hackathon/wandb/run-20210916_112128-20zmzt4y/files/wandb-summary.json
2021-09-16 11:21:52,188 INFO    Thread-8  :117553 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/JanataHack-NLP-Hackathon/wandb/run-20210916_112128-20zmzt4y/files/output.log
2021-09-16 11:21:54,188 INFO    Thread-8  :117553 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/JanataHack-NLP-Hackathon/wandb/run-20210916_112128-20zmzt4y/files/output.log
2021-09-16 11:21:54,642 DEBUG   SenderThread:117553 [sender.py:send():182] send: history
2021-09-16 11:21:54,644 DEBUG   SenderThread:117553 [sender.py:send():182] send: summary
2021-09-16 11:21:54,646 INFO    SenderThread:117553 [sender.py:_save_file():873] saving file wandb-summary.json with policy end
2021-09-16 11:21:54,647 DEBUG   SenderThread:117553 [sender.py:send():182] send: history
2021-09-16 11:21:54,647 DEBUG   SenderThread:117553 [sender.py:send():182] send: summary
2021-09-16 11:21:54,648 INFO    SenderThread:117553 [sender.py:_save_file():873] saving file wandb-summary.json with policy end
2021-09-16 11:21:54,814 DEBUG   SenderThread:117553 [sender.py:send():182] send: history
2021-09-16 11:21:54,815 DEBUG   SenderThread:117553 [sender.py:send():182] send: summary
2021-09-16 11:21:54,816 INFO    SenderThread:117553 [sender.py:_save_file():873] saving file wandb-summary.json with policy end
2021-09-16 11:21:54,840 DEBUG   SenderThread:117553 [sender.py:send():182] send: history
2021-09-16 11:21:54,840 DEBUG   SenderThread:117553 [sender.py:send():182] send: summary
2021-09-16 11:21:54,842 INFO    SenderThread:117553 [sender.py:_save_file():873] saving file wandb-summary.json with policy end
2021-09-16 11:21:55,189 INFO    Thread-8  :117553 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/JanataHack-NLP-Hackathon/wandb/run-20210916_112128-20zmzt4y/files/wandb-summary.json
2021-09-16 11:21:56,059 WARNING MainThread:117553 [internal.py:wandb_internal():145] Internal process interrupt: 1
2021-09-16 11:21:56,189 INFO    Thread-8  :117553 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/JanataHack-NLP-Hackathon/wandb/run-20210916_112128-20zmzt4y/files/output.log
2021-09-16 11:21:56,242 DEBUG   SenderThread:117553 [sender.py:send():182] send: telemetry
2021-09-16 11:21:56,242 DEBUG   HandlerThread:117553 [handler.py:handle_request():126] handle_request: poll_exit
2021-09-16 11:21:56,242 DEBUG   SenderThread:117553 [sender.py:send():182] send: exit
2021-09-16 11:21:56,243 INFO    SenderThread:117553 [sender.py:send_exit():291] handling exit code: 0
2021-09-16 11:21:56,243 INFO    SenderThread:117553 [sender.py:send_exit():293] handling runtime: 16
2021-09-16 11:21:56,244 INFO    SenderThread:117553 [sender.py:_save_file():873] saving file wandb-summary.json with policy end
2021-09-16 11:21:56,244 INFO    SenderThread:117553 [sender.py:send_exit():303] send defer
2021-09-16 11:21:56,244 DEBUG   SenderThread:117553 [sender.py:send_request():196] send_request: poll_exit
2021-09-16 11:21:56,245 DEBUG   HandlerThread:117553 [handler.py:handle_request():126] handle_request: defer
2021-09-16 11:21:56,245 INFO    HandlerThread:117553 [handler.py:handle_request_defer():143] handle defer: 0
2021-09-16 11:21:56,245 DEBUG   SenderThread:117553 [sender.py:send_request():196] send_request: defer
2021-09-16 11:21:56,245 INFO    SenderThread:117553 [sender.py:send_request_defer():312] handle sender defer: 0
2021-09-16 11:21:56,245 INFO    SenderThread:117553 [sender.py:transition_state():316] send defer: 1
2021-09-16 11:21:56,246 DEBUG   HandlerThread:117553 [handler.py:handle_request():126] handle_request: defer
2021-09-16 11:21:56,246 INFO    HandlerThread:117553 [handler.py:handle_request_defer():143] handle defer: 1
2021-09-16 11:21:56,315 DEBUG   SenderThread:117553 [sender.py:send_request():196] send_request: defer
2021-09-16 11:21:56,315 INFO    SenderThread:117553 [sender.py:send_request_defer():312] handle sender defer: 1
2021-09-16 11:21:56,315 INFO    SenderThread:117553 [sender.py:transition_state():316] send defer: 2
2021-09-16 11:21:56,315 DEBUG   SenderThread:117553 [sender.py:send():182] send: stats
2021-09-16 11:21:56,316 DEBUG   HandlerThread:117553 [handler.py:handle_request():126] handle_request: defer
2021-09-16 11:21:56,316 INFO    HandlerThread:117553 [handler.py:handle_request_defer():143] handle defer: 2
2021-09-16 11:21:56,316 DEBUG   SenderThread:117553 [sender.py:send_request():196] send_request: defer
2021-09-16 11:21:56,316 INFO    SenderThread:117553 [sender.py:send_request_defer():312] handle sender defer: 2
2021-09-16 11:21:56,316 INFO    SenderThread:117553 [sender.py:transition_state():316] send defer: 3
2021-09-16 11:21:56,317 DEBUG   HandlerThread:117553 [handler.py:handle_request():126] handle_request: defer
2021-09-16 11:21:56,317 INFO    HandlerThread:117553 [handler.py:handle_request_defer():143] handle defer: 3
2021-09-16 11:21:56,318 DEBUG   SenderThread:117553 [sender.py:send():182] send: summary
2021-09-16 11:21:56,319 INFO    SenderThread:117553 [sender.py:_save_file():873] saving file wandb-summary.json with policy end
2021-09-16 11:21:56,319 DEBUG   SenderThread:117553 [sender.py:send_request():196] send_request: defer
2021-09-16 11:21:56,319 INFO    SenderThread:117553 [sender.py:send_request_defer():312] handle sender defer: 3
2021-09-16 11:21:56,320 INFO    SenderThread:117553 [sender.py:transition_state():316] send defer: 4
2021-09-16 11:21:56,320 DEBUG   HandlerThread:117553 [handler.py:handle_request():126] handle_request: defer
2021-09-16 11:21:56,320 INFO    HandlerThread:117553 [handler.py:handle_request_defer():143] handle defer: 4
2021-09-16 11:21:56,320 DEBUG   SenderThread:117553 [sender.py:send_request():196] send_request: defer
2021-09-16 11:21:56,320 INFO    SenderThread:117553 [sender.py:send_request_defer():312] handle sender defer: 4
2021-09-16 11:21:56,350 DEBUG   HandlerThread:117553 [handler.py:handle_request():126] handle_request: poll_exit
2021-09-16 11:21:56,703 ERROR   SenderThread:117553 [internal_api.py:execute():133] 409 response executing GraphQL.
2021-09-16 11:21:56,703 ERROR   SenderThread:117553 [internal_api.py:execute():134] {"errors":[{"message":"Error 1062: Duplicate entry '454907-20zmzt4y' for key 'PRIMARY'","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2021-09-16 11:21:57,190 INFO    Thread-8  :117553 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/JanataHack-NLP-Hackathon/wandb/run-20210916_112128-20zmzt4y/files/output.log
2021-09-16 11:21:57,190 INFO    Thread-8  :117553 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/JanataHack-NLP-Hackathon/wandb/run-20210916_112128-20zmzt4y/files/wandb-summary.json
2021-09-16 11:21:58,217 ERROR   SenderThread:117553 [internal_api.py:execute():133] 409 response executing GraphQL.
2021-09-16 11:21:58,217 ERROR   SenderThread:117553 [internal_api.py:execute():134] {"errors":[{"message":"Error 1062: Duplicate entry '454907-20zmzt4y' for key 'PRIMARY'","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2021-09-16 11:22:00,916 ERROR   SenderThread:117553 [internal_api.py:execute():133] 409 response executing GraphQL.
2021-09-16 11:22:00,916 ERROR   SenderThread:117553 [internal_api.py:execute():134] {"errors":[{"message":"Error 1062: Duplicate entry '454907-20zmzt4y' for key 'PRIMARY'","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2021-09-16 11:22:01,326 INFO    WriterThread:117553 [datastore.py:close():281] close: /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/JanataHack-NLP-Hackathon/wandb/run-20210916_112128-20zmzt4y/run-20zmzt4y.wandb
2021-09-16 11:22:01,355 INFO    HandlerThread:117553 [handler.py:finish():683] shutting down handler
2021-09-16 11:22:01,355 ERROR   MainThread:117553 [internal.py:wandb_internal():157] Thread SenderThread:
Traceback (most recent call last):
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/sdk/internal/internal_api.py", line 130, in execute
    return self.client.execute(*args, **kwargs)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/vendor/gql-0.2.0/gql/transport/requests.py", line 39, in execute
    request.raise_for_status()
  File "/home/indika/anaconda3/lib/python3.7/site-packages/requests/models.py", line 953, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 409 Client Error: Conflict for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/apis/normalize.py", line 24, in wrapper
    return func(*args, **kwargs)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/sdk/internal/internal_api.py", line 1087, in upsert_run
    response = self.gql(mutation, variable_values=variable_values, **kwargs)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/sdk/lib/retry.py", line 102, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/sdk/internal/internal_api.py", line 135, in execute
    self.display_gorilla_error_if_found(res)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/sdk/internal/internal_api.py", line 152, in display_gorilla_error_if_found
    "Error while calling W&B API: {} ({})".format(err["message"], res)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/errors/term.py", line 63, in termerror
    **kwargs
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/errors/term.py", line 94, in _log
    click.echo(line, file=sys.stderr, nl=newline)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/click/utils.py", line 298, in echo
    file.write(out)  # type: ignore
BrokenPipeError: [Errno 32] Broken pipe

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/sdk/internal/internal_util.py", line 52, in run
    self._run()
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/sdk/internal/internal_util.py", line 102, in _run
    self._process(record)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/sdk/internal/internal.py", line 290, in _process
    self._sm.send(record)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/sdk/internal/sender.py", line 184, in send
    send_handler(record)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/sdk/internal/sender.py", line 198, in send_request
    send_handler(record)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/sdk/internal/sender.py", line 332, in send_request_defer
    self.debounce()
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/sdk/internal/sender.py", line 245, in debounce
    self._debounce_config()
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/sdk/internal/sender.py", line 252, in _debounce_config
    name=self._run.run_id, config=config_value_dict, **self._api_settings
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/apis/normalize.py", line 62, in wrapper
    six.reraise(CommError, CommError(message, err), sys.exc_info()[2])
  File "/home/indika/anaconda3/lib/python3.7/site-packages/six.py", line 718, in reraise
    raise value.with_traceback(tb)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/apis/normalize.py", line 24, in wrapper
    return func(*args, **kwargs)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/sdk/internal/internal_api.py", line 1087, in upsert_run
    response = self.gql(mutation, variable_values=variable_values, **kwargs)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/sdk/lib/retry.py", line 102, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/sdk/internal/internal_api.py", line 135, in execute
    self.display_gorilla_error_if_found(res)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/sdk/internal/internal_api.py", line 152, in display_gorilla_error_if_found
    "Error while calling W&B API: {} ({})".format(err["message"], res)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/errors/term.py", line 63, in termerror
    **kwargs
  File "/home/indika/anaconda3/lib/python3.7/site-packages/wandb/errors/term.py", line 94, in _log
    click.echo(line, file=sys.stderr, nl=newline)
  File "/home/indika/anaconda3/lib/python3.7/site-packages/click/utils.py", line 298, in echo
    file.write(out)  # type: ignore
wandb.errors.CommError: [Errno 32] Broken pipe
2021-09-16 11:22:01,360 INFO    MainThread:117553 [internal.py:handle_exit():76] Internal process exited
